experimental/packages/exporter-trace-otlp-http/test/browser/CollectorTraceExporter.test.ts

```diff
@@ -83,137 +84,221 @@ describe('OTLPTraceExporter - web', () => {
   });
 
   describe('export', () => {
+    let server: any;
     beforeEach(() => {
       collectorExporterConfig = {
         hostname: 'foo',
         attributes: {},
         url: 'http://foo.bar.com',
       };
+      server = sinon.fakeServer.create();
     });
 
-    describe('when "sendBeacon" is available', () => {
+    afterEach(() => {
+      server.restore();
+    });
+
+    const testSpansSendUsingXhr = () => {
+      const request = server.requests[0];
+      assert.strictEqual(request.method, 'POST');
+      assert.strictEqual(request.url, 'http://foo.bar.com');
+
+      const body = request.requestBody;
+      const json = JSON.parse(
+        body
+      ) as IExportTraceServiceRequest;
+      const span1 =
+        json.resourceSpans?.[0].scopeSpans?.[0].spans?.[0];
+
+      assert.ok(typeof span1 !== 'undefined', "span doesn't exist");
+      ensureSpanIsCorrect(span1);
+
+      const resource = json.resourceSpans?.[0].resource;
+      assert.ok(typeof resource !== 'undefined', "resource doesn't exist");
+      ensureWebResourceIsCorrect(resource);
+
+      ensureExportTraceServiceRequestIsSet(json);
+    };
+
+    describe('when "sendBeacon" is available and returns', () => {
       beforeEach(() => {
         collectorTraceExporter = new OTLPTraceExporter(
           collectorExporterConfig
         );
       });
 
-      it('should successfully send the spans using sendBeacon', done => {
-        collectorTraceExporter.export(spans, () => {
+      describe('"true"', () => {
+        it('should successfully send the spans', done => {
+          stubBeacon.returns(true);
+
+          collectorTraceExporter.export(spans, () => {
```

Comment:
Not that OTLPExporterBrowserBase in located in package otlp-exporter-base, I'm wondering if these tests could be move there too?